pgcli/main.py

```diff
@@ -926,9 +926,21 @@ def get_last_query(self):
         """Get the last query executed or None."""
         return self.query_history[-1][0] if self.query_history else None
 
+    # Ref: https://stackoverflow.com/questions/30425105/filter-special-chars-such-as-color-codes-from-shell-output
+    COLOR_PATTERN = re.compile(r'\x1b(\[.*?[@-~]|\].*?(\x07|\x1b\\))')
```

Comment:
We're doing the exact same thing when reporting an error in behave tests:

https://github.com/dbcli/pgcli/blob/5834259535e1cf5a75204305b0533ac5559ff344/tests/features/steps/wrappers.py#L13

See if maybe you can move it into a common method? or at least have a shared constant for this regexp.